examples/proc_GDS_pulse_capture.py

```diff
@@ -0,0 +1,121 @@
+"""
+Calculate actual beta as a function of pulse length
+===================================================
+Assuming the data is the capture of the pulse sequence as seen on the GDS
+oscilloscope, here the data is converted to analytic power, frequency filtered
+and the absolute is taken prior to integrating to return the beta value (us
+sqrt(W)). 
+"""
+import pyspecdata as psd
+import matplotlib.pyplot as plt
+from numpy import sqrt, array
+
+show_all = False
+with psd.figlist_var() as fl:
+    for filename, nodename, beta90_range, beta180_range in [
+        (
+            "240729_test_p90_pulse_calib.h5",
+            "pulse_calib_2",
+            (None, 34e-6),
+            (34e-6, None),
+        ),
+    ]:
+        d = psd.nddata_hdf5(
+            filename + f"/{nodename}",
+            directory=psd.getDATADIR(exp_type="ODNP_NMR_comp/test_equipment"),
+        )
+        beta_1 = []
+        beta_2 = []
```

Comment:
initiate list for the beta values of the 90 pulse and the 180 pulse

Reply:
This doesn't make sense, and it is over complicated because this should be for only one pulse, as we have discussed

Reply:
Also, I do not understand why there is a list here the best way to do that is to pull the ND shape, and then pop the direct dimension.

Also, it occurs to me that you can use the same code both to get the data that you need for your calibration, as well as to verify the calibration once it's done. In the first case, your indirect dimension would be the programmed pulse length, and in the second case, your indirect dimension would be the programmed beta value. The strategy I describe above enables that with the same code.

Reply:
(Play with an example in I python to see what I'm talking about0)